packages/mobile/App.svelte

```diff
@@ -14,7 +14,7 @@
         shouldBeDarkMode,
     } from '@core/app'
     import { onboardingProfile } from '@contexts/onboarding'
-    import { Route, ToastContainer } from './components'
+    import { Route, ToastContainer, DrawerManager } from './components'
```

Comment:
We can / should use the `@components` path alias. Might need to add a wildcard entry so that we can import from specific paths within `@components`.

Reply:
I dont use mobile path aliases anywhere because they dont work out of the box because we use the desktop folder to run mobile, so adding new path aliases means tweaking the setup locally to patch desktop to use mobile path aliases